packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceRefreshSObjects.test.ts

```diff
@@ -140,13 +134,11 @@ describe('ForceGenerateFauxClasses', () => {
       await checkSObjectsAndRefresh(projectPath);
 
       expect(existsSyncStub.calledWith(sobjectsPath)).to.be.true;
-      expect(
-        telemetryEventStub.calledWith(
-          'sObjectRefreshNotification',
-          { type: SObjectRefreshSource.StartupMin },
-          undefined
-        )
-      ).to.be.true;
+      expect(telemetryEventStub.callCount).to.equal(1);
+      const telemetryCallArgs = telemetryEventStub.getCall(0).args;
+      expect(telemetryCallArgs[0]).to.equal('sObjectRefreshNotification');
+      expect(telemetryCallArgs[1]).to.deep.equal({ type: SObjectRefreshSource.StartupMin });
+      expect(telemetryCallArgs[2]).to.equal(undefined);
```

Comment:
This unigration tests was being funky after the changes.  I simplified the expects and it works 🤷